hypervisor/include/common/ptdev.h

```diff
@@ -19,6 +19,11 @@ enum intx_ctlr {
 #define PTDEV_INTR_MSI		(1U << 0U)
 #define PTDEV_INTR_INTX		(1U << 1U)
 
+#define GPU_OPREGION_SIZE	0x5000U
+#define GPU_OPREGION_GPA	0x40880000U
+#define PCIR_ASLS_CTL		0xfcU /* Opregion start addr register */
```

Comment:
register offset in PCIe configuration space for Opregion base address.

Will this be better?
And what does ASLS mean?

Reply:
Hi, Eddie. The comment `register offset in PCIe configuration space for Opregion base address.` is better, I will refine the comments.

ASLS stands for "ASL Storage" from Bspec. https://github.com/torvalds/linux/blob/master/drivers/gpu/drm/i915/i915_reg.h#L347 Linux uses `ASLS` too.

Reply:
Make an explicit comment for readers to easily understand. We can assume ASL is a known term.

Reply:
OK, will refine the comment.